option_parser/src/lib.rs

```diff
@@ -167,6 +169,36 @@ impl OptionParser {
             .is_some()
     }
 
+    pub fn get_pci_device_function(
```

Comment:
Please add brief documentation about the inputs and the output format. Especially the input format.